### ISSUE TITLE
Snooze Confirmation Feature for Alarms

### ISSUE THREAD

**Issue by Tiberth** [vicolo-dev/chrono#152]
**Is your feature request related to a problem? Please describe.**
Sometimes I miss my alarm because I instinctively turn it off and go back to sleep. It would be helpful to have a feature where the alarm requires a second interaction to turn off completely.

**Describe the solution you'd like**
I would like to suggest adding a "Snooze Confirmation" feature to the timer app. This means that when the alarm goes off, instead of just turning it off with one tap, users would need to confirm the snooze action with a second tap, ensuring they are fully awake before dismissing the alarm. This feature should not replace the normal snooze function but rather be an option that users can set for specific alarms. Users should also be able to customize the duration of time they must confirm wakefulness before the alarm is dismissed completely.

**Describe alternatives you've considered**
One alternative could be adding a longer snooze time, but I believe a "Snooze Confirmation" feature would be more effective in ensuring users are fully awake before dismissing the alarm.

**Additional context**
This feature could greatly improve the user experience, especially for heavy sleepers or those prone to accidentally dismissing alarms.

And finally, I'd like to express my gratitude to you for this wonderful project. Thank you!

**Comment by AhsanSarwar45** [comment]
Yup got it, thanks for clarifying. That sounds like an interesting and pretty useful feature, will definitely consider it.
